packages/expo-medplum-polyfills/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@medplum/expo-medplum-polyfills",
```

Comment:
By convention, we leave out `medplum` from project names because it's implicit with the `@medplum` namespace.  So, `@medplum/expo-polyfills`